test/e2e/set/scyllacluster/scyllacluster_cleanup.go

```diff
@@ -96,14 +96,20 @@ var _ = g.Describe("ScyllaCluster", func() {
 
 		o.Expect(jobEvents).NotTo(o.BeEmpty())
 
+		tokenRingHash, err := utils.GetCurrentTokenRingHash(ctx, f.KubeClient().CoreV1(), sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		framework.Infof("Current token ring hash is %q", tokenRingHash)
```

Comment:
Why is this a valuable information? Should I be able to compare it with something? Or know if it's not the expected one?

Reply:
Each cleanup Job has ring hash annotation, the one Operator noticed and created a Job for. Annotations in artifacts should be the same as one printed here. In case of a failure, we can verify if job was stale or not.